docs/reference/buildx_prune.md

```diff
@@ -21,6 +21,18 @@ Remove build cache
 
 <!---MARKER_GEN_END-->
 
+## Description
+
+Clears the build cache of the selected builder.
+
+You can finely control what cache data is kept using:
+
+- The `--filter=until=<duration>` flag to keep images that have been used in
+  the last `<duration>` time.
+- The `--keep-storage=<space>` flag to keep `<space>` bytes of data in the cache.
```

Comment:
We should explain more what this `<space>` means in here as we don't expect people to write bytes. Eg. Common example would be `--keep-storage 5GB`. Same for the duration(what is probably seconds if no unit?).